### ISSUE TITLE
Windows-Default LAF has strange offset in PopupMenus

### ISSUE THREAD

**Issue by SeeYouSunday** [apache/netbeans#5697]
### Apache NetBeans version

Apache NetBeans 17

### What happened

Using the Default LAF "Windows" causes PopupMenus to have strange x-Offsets for every MenuItem. This seems to happen on every type of menu like the Toolbar-Menu of NetBeans itself but also for every JMenuItem in a JPopupMenu used in a NetBeans Platform Application.

I made a Screenshot in NetBeans highlighting the resulting gab:
![image](https://user-images.githubusercontent.com/122348235/226842410-507333dd-ead1-47a4-ac4f-5b6c6f8ca7b9.png)

This does not happen on other LAF implementations like Flat LAF.

### How to reproduce

1. Activate LAF "Windows" through Tools --> Options --> Appearance --> Look and Feel --> Prefered Look and Feel
2. Reboot NetBeans
3. Open any Toolbar Menus

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Windows 10  (22H2 Build 19045.2364)

### JDK

11

### Apache NetBeans packaging

Apache NetBeans provided installer, Apache NetBeans binary zip, Apache NetBeans platform

### Are you willing to submit a pull request?

No

**Comment by deinloft-eng** [comment]
Is there any solution for this strange offset (at windows LAF)?

**Comment by neilcsmith-net** [comment]
> Using the Default LAF "Windows"

The default LAF is FlatLaf.  Fixes for other LAF are welcome.

**Comment by deinloft-eng** [comment]
I've meant using the Default configuration has this strange offset...

If I use the "Metal" laf, this is fixed, but it appers other strange "spaces".

And I need to use the same laf as the picture above.

**Comment by SeeYouSunday** [comment]
> The default LAF is FlatLaf. Fixes for other LAF are welcome.

You are correct, sorry for that. Flat LAF does (the actual default LAF) does not have this problem. But the LAF "Windows" does.

**Comment by myroch** [comment]
any workaround for this?
